### DISCUSSION TITLE
Negative edge-triggered sub-circuito clock input.

### DISCUSSION THREAD

**Discussion by RicardoKers** [hneemann/Digital#803] [Q&A] [answered]
Is it possible put the small circle on the clock input signal of a sub-circuit? I need signalize a negative edge-triggered device.

**Comment by hneemann** [comment, marked as answer]
No! But you can name the clock like "~CLK". This way it looks like this:

![zz](https://user-images.githubusercontent.com/7190563/128130156-a65fa3fa-7de5-4d0b-9bbf-9d36bb2ee58e.png)

However, it is possible to create a custom shape, but this is quite complicated.

**Reply by RicardoKers** [reply]
The custom shape is a good idea and solve my problem, thank you.
I take this opportunity to thank you for the great work with the simulator. It's a great tool.
